garak/buffs/encoding.py

```diff
@@ -16,10 +16,14 @@ class Base64(Buff):
     def transform(
         self, attempt: garak.attempt.Attempt
     ) -> Iterable[garak.attempt.Attempt]:
-        b64_attempt = b64encode(str.encode(attempt.prompt))
+        # transform receives a copy of the attempt should it modify the prompt in place?
```

Comment:
if we constrain buffing to something that happens at attempt/prompt init, yes

since the plan for conversations to actually be linked lists of attempts then buffing each garak-side turn in a conversation will involve minting new attempts, so this fits